src/models/topic.ts

```diff
@@ -78,16 +82,12 @@ export class KafkaTopicTreeItem extends vscode.TreeItem {
 
   checkMissingAuthorizedOperations(resource: KafkaTopic): KafkaTopicOperation[] {
     // operations we support via view/item actions that require authorization
-    const interestingAuthz: KafkaTopicOperation[] = ["READ", "DELETE", "ALTER_CONFIGS"];
+    const interestingAuthz: KafkaTopicOperation[] = ["READ", "DELETE"];
 
     for (const op of interestingAuthz) {
       if (resource.operations.includes(op)) {
         // Convert to "authzRead", "authzDelete", etc. for context flags to hang context-sensitive commands off of (see package.json)
-        const operationTitleCase = op
-          .toLowerCase()
-          .replace(/_/g, " ") // replace underscores with spaces
-          .replace(/^\w|\s\w/g, (c) => c.toUpperCase()) // convert to title case
-          .replace(/\s/g, ""); // remove spaces
+        const operationTitleCase = op.toLowerCase().replace(/^\w/, (c) => c.toUpperCase());
```

Comment:
this reverts an old change I made for configure topic (adding "ALTER_CONFIGS"). Was that intentional?